R/deparse.R

```diff
@@ -703,7 +703,7 @@ default_deparse <- function(x, lines = new_lines()) {
   lines$get_lines()
 }
 
-sexp_deparse <- function(x, lines = new_lines()) {
+sexp_deparse <- function(x, lines = new_lines(), max_len = 5L) {
```

Comment:
This function should validate `max_len`.

Reply:
Done